<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Title-template</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="connectBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>331</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="connectionStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>311</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string>Нет подключения</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>227</width>
       <height>205</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QRadioButton" name="regButton">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Регистрация</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="authButton">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Авторизация</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLineEdit" name="loginLine">
              <property name="placeholderText">
               <string>Логин</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pswdLine">
              <property name="placeholderText">
               <string>Пароль</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QPushButton" name="execButton">
            <property name="text">
             <string>Выполнить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="connectButton">
            <property name="text">
             <string>Подключиться</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="answerBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>313</width>
      <height>188</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>14</y>
       <width>289</width>
       <height>130</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Ваш ответ:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Номер варианта:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="taskLine"/>
          </item>
          <item>
           <widget class="QLineEdit" name="answerLine"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="sendTaskButton">
        <property name="text">
         <string>Отправить на проверку</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelAnswerStatus">
          <property name="text">
           <string>Статус ответа:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="answerStatusLabel">
          <property name="text">
           <string>Неизвестно</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="rerollGraph">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>221</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Сгенерировать новый граф</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>50</y>
      <width>525</width>
      <height>525</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="task19Variant">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>250</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="task19answer">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>280</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="task19Send">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>310</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="task19Result">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>350</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Неизвестно</string>
    </property>
   </widget>
   <widget class="QLabel" name="task19Prufer">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>370</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <action name="action">
   <property name="text">
    <string>Авторизация</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
